Donor/Entities/Donor.cs

```diff
@@ -15,6 +19,13 @@ public class Donor
         public string MobileNumber { get; set; } = string.Empty;
         public string Nationality { get; set; } = string.Empty;
         public string Gender { get; set; } = string.Empty;
+        
+        public PreferredContactMode PreferredContact { get; set; }
+        public ICollection<DonationPreference> DonationPreferences { get; set; } = new List<DonationPreference>();
```

Comment:
Instead of  a collection of DonationPreference
it should be  a collection of Organ plus the OrganId property.
This is related with my previous comment